styles/daosList.scss

```diff
@@ -4,7 +4,7 @@
   display: grid;
   grid-gap: 50px;
   grid-template-columns: 1fr;
-  place-items: start center;
+  place-items: start;
```

Comment:
Can you doublechek it with design team, please? I believe it was request from Max to center cards.

Reply:
Such fix will brake cards centering in grid. This should be confirmed be design team definitely.
In general, this issue "Single DAO is not aligned on the left side" doesn't look like as issue at all.
On some display resolutions we have centered DAO even if it isn't single https://ibb.co/FBN1CHB